test/token/ERC721/ERC721Full.test.js

```diff
@@ -138,6 +138,14 @@ contract('ERC721Full', function ([
       });
     });
 
+    describe('tokensOfOwner', function () {
+      it('returns total tokens of owner', async function () {
+        const tokenIds = await this.token.tokensOfOwner(owner);
```

Comment:
We should also be checking that the array length is exactly as expected.

I think we can use Chai's [`eql`](https://www.chaijs.com/api/bdd/#method_eql) assertion to abbreviate that and the next two lines.

```javascript
tokenIds.should.eql([firstTokenId, secondTokenId]);
```

Reply:
Ids are returned as bignumber. Is there any related assertion for that?

Reply:
Hm, I see the problem. Then I'd say just add the check for length. I think the following should work.

```suggestion
        const tokenIds = await this.token.tokensOfOwner(owner);
        tokenIds.length.should.equal(2);
```